internal/generate/olm-catalog/csv_go_test.go

```diff
@@ -16,16 +16,14 @@ package olmcatalog
 
```

Comment:
Most changes to this file are cosmetic except for setting `bundleGenerator.noUpdate`, which prevents the generator from reading a CSV to update from so we can check a newly-generated CSV against the expected CSV in `testdata`.